src/index.html

```diff
@@ -7,12 +7,64 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymus"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article
+      class="product-card"
+      data-qa="card"
+    >
+      <img
+        class="product-card__img"
+        src="../../layout_product-cards/src/images/imac.jpeg"
+        alt="imac-img"
+      />
+
+      <div class="product-card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
+
+      <div class="product-card__code">Product code: 195434</div>
```

Comment:
```suggestion
      <p class="product-card__code">Product code: 195434</p>
```
Consider leveraging semantic tags for the text